.github/workflows/screenshots.yml

```
@@ -30,26 +27,20 @@ jobs:
     - name: Compile the App
       run: bundle exec fastlane build_screenshots
 
-    - name: Archive App
+    - name: Archive Build Products
       uses: actions/upload-artifact@v4
       with:
-        name: screenshot-app
-        path: fastlane/DerivedData/Build/Products/Debug-iphonesimulator/WooCommerce.app
-
-    - name: Archive Runner
-      uses: actions/upload-artifact@v4
-      with:
-        name: screenshot-runner
-        path: fastlane/DerivedData/Build/Products/Debug-iphonesimulator/WooCommerceScreenshots-Runner.app
+        name: screenshot-build-products
+        path: fastlane/DerivedData/Build/Products/Debug-iphonesimulator/
```

Probably not worth it for the storage reclaim per se, but maybe worth it to make it clearer that these are discardable adding something like:
```suggestion
        path: fastlane/DerivedData/Build/Products/Debug-iphonesimulator/
        retention-days: 1
```

---

Good idea, updated in https://github.com/woocommerce/woocommerce-ios/pull/15822/commits/f02111c8e87d7a95c9b0d44c7213b471a210d5b9.